http3/client.go

```diff
@@ -44,6 +44,7 @@ type roundTripperOpts struct {
 	MaxHeaderBytes     int64
 	AdditionalSettings map[uint64]uint64
 	StreamHijacker     func(FrameType, quic.Connection, quic.Stream) (hijacked bool, err error)
+	UniStreamHijacker  func(FrameType, quic.Connection, quic.ReceiveStream) (hijacked bool, err error)
```

Comment:
This is not a frame type, it's a stream type. We should have a separate type for this.

Reply:
Yes, I was misunderstanding. I fixed it.